docs/content/ActionMenu.mdx

```diff
@@ -117,58 +117,53 @@ You can choose to have a different _anchor_ for the Menu depending on the applic
   <ActionMenu.Button>Open column menu</ActionMenu.Button>
 
   <ActionMenu.Overlay>
+    <ActionList showDividers headingProps={{title: 'Live query'}}>
```

Comment:
Need to update example

Ditto as ActionList, we should change the heading to `With Heading` or `divided into sections`